src/style.css

```diff
@@ -2,2 +1,78 @@
+html,
+body,
+.header,
+.img-link,
+.nav-link,
+.nav-list {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', 'Arial', sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.img-link {
+  display: flex;
+  text-decoration: none;
+}
+
+.nav-list {
+  list-style: none;
```

Comment:
Avoid using tag names for styling. This can lead to unexpected results if new elements of the same type are added later. Instead, style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.